deploy.sh

```diff
@@ -119,10 +119,13 @@ echo "::debug::Removed old local ZIP Archive."
 
 # 4) Start the CodeDeploy
 function getActiveDeployments() {
-    aws deploy list-deployments \
+    if ! aws deploy list-deployments \
         --application-name "$INPUT_CODEDEPLOY_NAME" \
         --deployment-group-name "$INPUT_CODEDEPLOY_GROUP" \
-        --include-only-statuses "Queued" "InProgress" |  jq -r '.deployments';
+        --include-only-statuses "Queued" "InProgress" |  jq -r '.deployments'; then
+        echo -e "${RED}Failed deployment detected (Failed ListDeployments see error above)."
```

Comment:
I don't know if our deployment technically failed. It started, but we can't monitor it. So perhaps we need to tweak this message slightly.

Reply:
So something like:
`Failed deployment monitoring (Failed ListDeployments see error above).`

Reply:
Yeah I'm hoping AWS runs an endpoint with 1-1 to a permission. So we could just say we are missing ListDeployments permission and thus this call failed.

Reply:
Gotcha. For now how do we want to handle this issue?

Good to just have the error changed to `Failed deployment monitoring (Failed ListDeployments see error above).` for now?

Reply:
Yeah I guess thats safest bet since we don't know if an issue is permission or something else.

Reply:
Maybe even an orange/warning message prior that deployment may be executing. We just can't tell.

Reply:
Okay so basically a line before the Failed. Like this:

```
echo -e "${ORANGE}Warning deployment may be running."
echo -e "${RED}Failed deployment monitoring (Failed ListDeployments see error above)."
```

Reply:
I'm torn now. Now its a warning which I want, but its the expected behavior. Let me sleep on this and brainstorm.

Reply:
How'd the sleep go with this on your mind? 😆

Reply:
```bash
echo -e "${ORANGE}Deployment may still be executing."
echo -e "${RED}Failed monitoring deployment (ListDeployments API call failed)."
```

How do you think about this?

Reply:
That works for me.

Reply:
I made this change.